using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -325,6 +325,73 @@ scheme://authority/path?TS=timestamp&MCMID=ecid&MCORGID=ecorgid@AdobeOrg#fragmen
 If your application uses more complicated URLs, such as Angular URLs, we recommend that you use getUrlVariables.
 {% endhint %}
 {% endtab %}
+
+{% tab title="Cordova" %}
+
+### appendVisitorInfoForURL
+
+This API appends Adobe visitor information to the query component of the specified URL.
+
+If the specified URL is nil or empty, it is returned as is. Otherwise, the following information is added to the query component of the specified URL.
+
+* The `adobe_mc` attribute is a URL encoded list that contains:
+  * `MCMID` - Experience Cloud ID \(ECID\)
+  * `MCORGID` - Experience Cloud Org ID
+  * `MCAID` - Analytics Tracking ID \(AID\), if available from the [Analytics extension](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-analytics/analytics-api-reference#gettrackingidentifier)
+  * `TS` - A timestamp taken when this request was made
+* The optional `adobe_aa_vid` attribute is the URL-encoded Analytics Custom Visitor ID \(VID\), if previously set in the [Analytics extension](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-analytics/analytics-api-reference#setidentifier).
+
+#### Cordova
+
+**Syntax**
+
+```jsx
+ACPIdentity.appendVisitorInfoForUrl = function(url, success, fail);
+```
+
+* _url_ _\(String\)_ is the URL to which the visitor information needs to be appended. If the visitor information is nil or empty, the URL is returned as is.
+* _success_ is a callback containing the provided URL with the visitor information appended if the `appendVisitorInfoForUrl` API executed without any errors.
+* _fail_ is a callback containing error information if the  `appendVisitorInfoForUrl` API was executed with errors.
+
+**Example**
+
+```jsx
+ACPIdentity.appendVisitorInfoForUrl("https://example.com", function(handleCallback) {
+  console.log("AdobeExperenceSDK: Url with Visitor Data = " + handleCallback);
+}, function(handleError) {
+  console.log("AdobeExperenceSDK: Failed to append URL : " + handleError);
+});
+```
+
+{% hint style="info" %}
+This API is designed to handle the following URL formats:
+
+```text
+scheme://authority/path?query=param#fragment
+```
+
+In this example, the Adobe visitor data is appended as:
+
+```text
+scheme://authority/path?query=param&TS=timestamp&MCMID=ecid&MCORGID=ecorgid@AdobeOrg#fragment
+```
+
+Similarly, URLs without a query component:
+
+```text
+scheme://authority/path#fragment
+```
+
+The Adobe visitor data is appended as:
+
+```text
+scheme://authority/path?TS=timestamp&MCMID=ecid&MCORGID=ecorgid@AdobeOrg#fragment
+```
+
```

Comment:
Looks like this is missing the {% hint style="info" %} start tag.

Reply:
The hint start tag is at line 366. I did spot an issue with the link at 391 which I will fix.